### PR TITLE
fix(ui): API endpoints to fetch elastic search data on settings page.

### PR DESCRIPTION
**Pull request by aniketkatkar97**
### Describe your changes:

Fixes errors on ElasticSearch re-index settings page.

I worked on the changes to use the newly created APIs for fetching elasticSearch re-indexing data

https://user-images.githubusercontent.com/51777795/230378895-4fd0993f-1eb7-4258-98bb-9e8a25d34dff.mov

#
### Type of change:

- [x] Bug fix

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/pages/ElasticSearchIndexPage/ElasticSearchReIndexPage.component.tsx [hunk 1]
```diff
@@ -191,24 +193,26 @@ const ElasticSearchIndexPage = () => {
...
                           'label.status'
                         )}:`}</span>
-                        <span className="m-l-xs">
-                          <Space size={8}>
-                            {batchJobData?.status && (
+
+                        <Space align="center" className="m-l-xs" size={8}>
+                          {batchJobData?.status &&
+                            (batchJobData.status === Status.Active ? (
+                              <Loader size="x-small" />
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/ElasticSearchIndexPage/ElasticSearchReIndexPage.component.tsx:200]: you can put this in this method right? getStatusResultBadgeIcon

File: openmetadata-ui/src/main/resources/ui/src/pages/ElasticSearchIndexPage/ElasticSearchReIndexPage.component.tsx [hunk 2]
```diff
@@ -355,30 +359,32 @@ const ElasticSearchIndexPage = () => {
...
                         <span className="text-grey-muted">{`${t(
                           'label.status'
                         )}:`}</span>
-                        <span className="m-l-xs">
-                          <Space size={8}>
-                            {streamJobData?.status && (
+                        <Space align="center" className="m-l-xs" size={8}>
+                          {streamJobData?.status &&
+                            (streamJobData.status === Status.Active ? (
+                              <Loader size="x-small" />
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/ElasticSearchIndexPage/ElasticSearchReIndexPage.component.tsx:370]: same as above

**Review by chirag-madlani — APPROVED** [review]
UI side looks good 🚀

**Review by mohityadav766 — APPROVED** [review]

**Comment by mohityadav766** [comment]
@aniketkatkar97 will need some work, currently we are not getting any error and context that are sent with jobData

**Pull request merged**